internal/cmd/root.go

```diff
@@ -32,6 +32,7 @@ func init() {
 		fmt.Fprintf(os.Stderr, "error binding token flag: %s", err)
 	}
 	rootCmd.PersistentPostRun = func(cmd *cobra.Command, args []string) {
+		viper.Set("no-multiple-token-sources-warning", false)
 		settings.PersistChanges()
```

Comment:
Yeah, I wasn't sure about the motivations for `PersistChanges` so I opt to leave it as a draft, just turn `no-multiple-token-sources-warning` back to `false` and wait for more info.

Reply:
@athoscouto changed the way settings behave here -> https://github.com/chiselstrike/turso-cli/pull/392
Now we just save the settings once at the end.

Reply:
Why are we binding the flag to viper? 
This will effectively make the flag vaalue persistent throughout command calls.
Not to be confused with persistent flags, which are flags that are inherited by subcommands.

Reply:
Using cobra's persistent flags is different from biding it to viper.
See [this](https://github.com/spf13/cobra/blob/main/user_guide.md#bind-flags-with-config).

When you do:
``` 
	rootCmd.PersistentFlags().StringP("author", "a", "YOUR NAME", "author name for copyright attribution")
```

You're using cobra's persistent flag.

When you do:
```
	viper.BindPFlag("author", rootCmd.PersistentFlags().Lookup("author"))
```

You're biding it to viper, and since we persist viper configs, this will make the flag value to be durable between command calls. It seems that we don't want this for `no-multiple-token-sources-warning`.

Reply:
The nomenclature is confusing:
- Persistent flags are flags shared with subcommands.
- Biding it to viper makes the flag value (not the flag itself) persistent/durable between command calls.

Reply:
If we don't bind it to viper, I can not use `viper.GetBool("no-multiple-token-sources-warning")` anymore, right?

What would be the best way to access it in another command? store it in a var?

Reply:
Exactly what @StefanoSaffran said.

Reply:
If you don't bind it to viper, you won't be able to get it from viper with GetBool. But why not getting it from the flag variable itself?

Reply:
Anyway, feel free to get this merged since it solves a functional problem. We can iterate on the interaction between flags and config later.

Reply:
added it to a var and removed the bind.